packages/mds-orm/types.ts

```diff
@@ -0,0 +1,23 @@
+import { InsertResult } from 'typeorm'
+
+export type AsEntity<T> = {
```

Comment:
What does this type do exactly? I looked for an example of its use but I don't think it's used anywhere in this PR.

Reply:
Basically, it maps object properties from `T[P] | undefined` (i.e. _optional_) to `T[P] | null` which is how they are returned from a database query. I ended up not using it though because the type mapping from persistence to domain models ended up being a little more complex.